spec/generators/rails-template.rb

```diff
@@ -133,7 +135,7 @@ def wood_type
     class Wizard < ApplicationRecord
       validates :name, length: { minimum: 5 }, presence: true
       # simulate conditional validation
-      validates :parent_email, presence: true, if: -> { false }
```

Comment:
Sorbet was complaining about `-> { false }` and kept trying to downgrade this file so I changed it to be a symbol that references a method.